src/error.js

```diff
@@ -0,0 +1,15 @@
+const getError = (requestedPath) => {
+  if (!requestedPath.startsWith('file')) {
+    return [400, 'The path to files must start with /file'];
```

Comment:
The error message should be updated to 'The path to files must start with /file/' for consistency and clarity.